source/Halibut.Tests/Support/BackwardsCompatibility/LatestClientAndPreviousServiceVersionBuilder.cs

```diff
@@ -302,13 +303,17 @@ public TService CreateClient<TService>(Action<ServiceEndPoint> modifyServiceEndp
 
             public void Dispose()
             {
-                cancellationTokenSource?.Cancel();
-                Client.Dispose();
-                runningOldHalibutBinary.Dispose();
-                HttpProxy?.Dispose();
-                PortForwarder?.Dispose();
-                disposableCollection.Dispose();
-                cancellationTokenSource?.Dispose();
+                var logger = new SerilogLoggerBuilder().Build().ForContext<ClientAndService>();
+                logger.Information("Dispose called");
+                Action<Exception> logError = e => logger.Warning(e, "Ignoring error in dispose");
+                
+                Try.CatchingError(() => cancellationTokenSource?.Cancel(), logError);
+                Try.CatchingError(Client.Dispose, logError);
+                Try.CatchingError(runningOldHalibutBinary.Dispose, logError);
+                Try.CatchingError(() => HttpProxy?.Dispose(), logError);
+                Try.CatchingError(() => PortForwarder?.Dispose(), logError);
+                Try.CatchingError(() => disposableCollection.Dispose(), logError);
+                Try.CatchingError(() => cancellationTokenSource?.Dispose(), logError);
```

Comment:
We don't care if an exception is raised in the dispose, so don't fail a test if an exception is raised in dispose.